docs/resources/widgets/pages/how-tos.md

```diff
@@ -0,0 +1,481 @@
+---
+title: How-to
+sidebar_position: 2
+toc_max_heading_level: 5
+---
+
+#
+
+FlutterFlow allows you to easily create new pages using the Page Selector
+button, which is available in the Widget Tree menu (Tab+2). This will help you
+quickly start and add new pages to your app.
```

Comment:
It feels a bit strange to me that were talking about creating an empty page after we've already shown the user how to configure the page

Reply:
Maybe we move the configurations to be a part of this how to guide